src/utils/schema-version-check.js

```diff
@@ -10,14 +10,45 @@
 
 export const CURRENT_SUPPORTED_SCHEMA = {
   major: 0,
-  minor: 2,
+  minor: 3,
   patch: 0,
 };
 
 export function combineVersions({ major, minor, patch }) {
   return [major, minor, patch].join('.');
 }
 
+/**
+ * Compares two version strings
+ * @param {string} v1 - first version to compare
+ * @param {string} v2 - second version to compare
+ * @returns {Number<-1|0|1>}
+ */
+export function compareVersions(v1, v2) {
+  const v1parts = v1.split('.');
```

Comment:
We should probably convert the parts into integers, even if the JS comparison of integer strings happens to work in the way that you might expect.